conf/application.conf

```diff
@@ -216,3 +216,19 @@ internal-auth.token = "<INSERT-VALUE>"
 redirectAllowedHosts = ["www.gov.uk"]
 
 fileMaxUploadedSizeMB = 145
+
+# caching config
+formTemplateCache {
+    expiry = 1 nanoseconds
```

Comment:
Why have a cache of 1 nanoseconds? 

Is the idea to override this in prod only?